app/src/main/java/com/akash/newsapp/NewsApplication.kt

```diff
@@ -32,8 +33,8 @@ class NewsApplication : Application() {
                 }
             }
         }
+        Glimpse.init(this)
```

Comment:
@aniketbhoite Move Glimpse initialisation in oncreate() method.

Reply:
Done